activation/listeners.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !no_listeners
```

Comment:
go build tags are not namespaced so there is always risk that it could conflict when another project uses the same name.

As such I would prefer something more like `go_systemd_no_listeners` or even `coreos_go_systemd_no_listeners`